README.md

```diff
@@ -763,6 +766,32 @@ syntax assuming that the private registry is hosted at "your.registry.org".
 helm upgrade --install deployment-manager --set "manager.image.repository=your.registry.com/wind-river/cloud-platform-deployment-manager" wind-river-cloud-platform-deployment-manager-2.0.10.tgz
 ```
 
+# Deployment Manager as System application
+
+***Note:*** Applicable only for WRCP
```

Comment:
Fluxcd application can also be applied to StarlingX

Reply:
Added a absolute path and reason also.